src/components/StepIndicator.module.scss

```diff
@@ -0,0 +1,86 @@
+@use "../styles/colors" as *;
+
+$step-number-size: 2.0625rem;
+
+.container {
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+  height: 10.75rem;
+  padding: 2rem 0 6.6875rem;
+  background-image: url("../assets/images/bg-sidebar-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.step {
+  display: flex;
+  flex-direction: column;
+  min-height: $step-number-size;
+}
+
+.stepNumber {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 100%;
+  border: 1px solid white;
+  color: white;
+  width: $step-number-size;
+  height: $step-number-size;
+  font-weight: 700;
+  translate:
+    color 100ms ease-in,
+    background-color 100ms ease-in;
+
+  &.active {
+    color: $marine-blue;
+    background-color: $light-blue;
+    border-width: 0;
+  }
+}
+
+.stepId {
+  display: none;
+  text-transform: uppercase;
+  color: $pastel-blue;
+}
```

Comment:
it looks a bit odd to have other css props if display is set to none, same for stepName